l10n_ch_import_cresus/wizard/l10n_ch_import_cresus.py

```diff
@@ -137,194 +71,150 @@ def _parse_csv(self):
-                    analytic_code_inverted = line_cresus['analytic_account']
-            inverted_default_value.update({'line_id/account_tax_id':
-                                           tax_code_inverted,
-                                          'line_id/analytic_account_id':
-                                           analytic_code_inverted})
-            new_openerp_data.append(inverted_default_value)
-
-        return new_openerp_data
-
-    @api.multi
-    def _load_data(self, data):
-        """Function that does the load of parsed CSV file.
-
-        If will log exception and susccess into the report fields.
+        """ split accounting lines where needed
 
-        :param data: CSV file content (list of data list)
+            Cresus writes one csv line per move when there are just two lines
+            (take some money from one account and put all of it in another),
+            and uses ellipses in more complex cases. What matters is the pce
+            label, which is the same on all lines of a move.
         """
-        # Change data from dict to list of array
-        data_array = []
-        for data_item_dict in data:
-            data_item = []
-            for item in self.HEAD_ODOO:
-                data_item.append(data_item_dict[item])
-            data_array.append(data_item)
-        try:
-            res = self.env['account.move'].load(self.HEAD_ODOO,
-                                                data_array)
+        journal_id = self.journal_id.id
+        previous_pce = None
+        previous_date = None
+        lines = []
+        for self.index, line_cresus in enumerate(data, 1):
+            if previous_pce is not None and previous_pce != line_cresus['pce']:
+                yield self.make_move(
+                    lines,
+                    date=previous_date,
+                    ref=previous_pce,
+                    journal_id=journal_id)
+                lines = []
+            previous_pce = line_cresus['pce']
+            previous_date = line_cresus['date']
+
+            recto_amount = float(line_cresus['amount'].replace('\'', '')
+                                                      .replace(' ', ''))
+            verso_amount = 0.0
+            if recto_amount < 0:
+                recto_amount, verso_amount = 0.0, -recto_amount
+            if line_cresus['debit'] != '...':
+                account = self._find_account(line_cresus['debit'])
+                tax = self._find_tax(line_cresus['typtvat'], account)
+                analytic_account = self._find_analytic_account(
+                    line_cresus['analytic_account'], account)
+                lines.append(self.make_line(
+                    recto_amount, verso_amount,
+                    account_id=account.id,
+                    partner_id=False,
+                    name=line_cresus['ref'],
+                    tax_line_id=tax.id,
+                    analytic_account_id=analytic_account.id))
```

Comment:
don't you want to pass the account to the method "make_line" so you can define all those aruments in it ?